docs/snippets/angular/button-story-decorator.ts.mdx

```diff
@@ -0,0 +1,26 @@
+```ts
+// button.stories.ts
+
+import { ButtonComponent } from './button.component';
+
+export const Primary: Story<ButtonComponent> = () => ({
+  component: ButtonComponent,
+  moduleMetadata: {
+    declarations: [ButtonComponent],
+  },
+  template: '<app-button label="Submit"></app-button>',
+});
+
+// Note: A template with the component's selector is required in the component story for the decorators to take place.
```

Comment:
@profanis i'm ok with the comment, but push it up to be next to the story and we probably could replace it with:
"To see the decorator applied to your component's stories you'll need to include the template key with the component's selector."

Reply:
This is done 😊

Thanks for the suggestion

Reply:
@jonniebigodes I saw that tests failed in CI. I `yarn test` before push and all the tests were green. Obviously, I miss something here :( 
Can you please advise? Many thanks

Reply:
You need to update the snapshots. `yarn test --core --update`

Reply:
Still the same

```
● Test suite failed to run

    Cannot find module 'downshift' from 'lib/ui/src/components/sidebar/Search.tsx'

    Require stack:
      lib/ui/src/components/sidebar/Search.tsx
      lib/ui/src/components/sidebar/Sidebar.tsx
      lib/ui/src/containers/sidebar.tsx
      lib/ui/src/app.tsx
      lib/ui/src/index.tsx
      lib/ui/src/__tests__/index.js

      4 | import { styled } from '@storybook/theming';
      5 | import { Icons } from '@storybook/components';
    > 6 | import Downshift, { DownshiftState, StateChangeOptions } from 'downshift';
        | ^
      7 | import Fuse, { FuseOptions } from 'fuse.js';
      8 | import React, { useEffect, useMemo, useRef, useState, useCallback, FunctionComponent } from 'react';
      9 | 

      at Resolver.resolveModule (node_modules/jest-cli/node_modules/jest-resolve/build/index.js:307:11)
      at Object.<anonymous> (lib/ui/src/components/sidebar/Search.tsx:6:1)


Test Suites: 1 failed, 134 passed, 135 total
Tests:       1555 passed, 1555 total
Snapshots:   449 passed, 449 total
Time:        28.164 s, estimated 42 s
```

Reply:
Did you update from `next` and `yarn install` first?

Reply:
🤦‍♂️🤦‍♂️🤦‍♂️ I didn't `yarn install`. 
Now the tests are green. 

```
Test Suites: 135 passed, 135 total
Tests:       1556 passed, 1556 total
Snapshots:   449 passed, 449 total
Time:        28.751 s
Ran all test suites in 7 projects.
```